### PR TITLE
Added a "Your week in PostHog" email template (for app)

### PR DESCRIPTION
**Pull request by corywatilo**
Made for @raquelmsmith 

Created it here since we have the tooling set up for it. Idea would be to have us just export the HTML into wherever we need it.

image

### CODE REVIEW

**Comment by raquelmsmith** [comment]
Noice! Looks great. How do I export the HTML?

**Review by raquelmsmith — APPROVED** [review]

**Comment by corywatilo** [comment]
@smallbrownbike Can you explain the best way to get the raw HTML for this?

(Or maybe you can pull it from the preview email I sent you, @raquelmsmith?)

The two things I'd obviously change from here:

1. Visit the PostHog community → Visit the PostHog app
2. Change the line at the bottom to something about turning off that type of email

**Comment by smallbrownbike** [comment]
@raquelmsmith You can peep the `react-email` docs for exporting to HTML [here](https://react.email/docs/cli#email-export)

TLDR; Run `yarn email export`
